### DISCUSSION TITLE
How to hide the CBC solver output in terminal and store it in a file instead?

### DISCUSSION THREAD

**Discussion by dassouki** [coin-or/pulp#339] [Q&A] [answered]
As the title suggests, what is the correct way to not show the CBC Solver output and store it in a file instead.

**Comment by pchtsp** [comment, marked as answer]
In the docs are the possible arguments one can pass the cbc solver that comes with pulp: 
https://coin-or.github.io/pulp/technical/solvers.html#pulp.apis.PULP_CBC_CMD

if you pass the `logPath` argument with the path of the log file, it should write to it.

**Reply by dassouki** [reply]
Thanks,

`prob.solve(lp.PULP_CBC_CMD(logPath=r'path.lp'))` which worked.

Note that when I defined the problem and added a `lp.LogPath = r'path.lp' `and `prob.logPath=r'path.lp'`. Both didn't return any errors / issues.
